spotbugsTestCases/src/java/bugIdeas/Ideas_2011_08_05.java

```diff
@@ -49,8 +49,6 @@ public int test4OK() {
 
     @NoWarning("RCN")
     public int test5(@Nonnull String x) {
-        if (x == null)
```

Comment:
This is a test idea example, why is this changed? It makes no sense.

Reply:
As the parameter is marked by *Nonnull*, *Traute* inserts a *null*-check there and the test fails like below:  

```
edu.umd.cs.findbugs.DetectorsTest > testAllRegressionFilesJavac FAILED
    java.lang.AssertionError: Unexpected bugs (1):
    unexpected RCN Redundant null check at Ideas_2011_08_05.java:[line 52]
        at org.junit.Assert.fail(Assert.java:88)
        at edu.umd.cs.findbugs.DetectorsTest.checkForUnexpectedBugs(DetectorsTest.java:142)
```  

Removed the explicit check. The tests pass now